teps/0092-scheduling-timeout.md

```diff
@@ -211,17 +213,17 @@ field would be deprecated once this feature is beta.
 ```yaml
 spec:
   timeouts:
-    scheduling: "5m" # this is the new option
-    execution: "10m" # this option would have the same functionality as the existing `timeout` field
+    scheduling: "5m" # optional, if specified, from startTime to scheduledTime
```

Comment:
I can't comment on the sections you didn't update, but can you flesh out the "design details" section a bit more on what happens if the various timeouts are set to 0? In addition, the "PipelineRun" section also needs to be updated to reflect the "execution" timeout in addition to the "scheduling" timeout, and the "The field in TaskRuns that represents the existing timeout" section needs to be updated to reflect the fact that the execution timeout is different from the existing timeout.

Lastly, on line 211, "The previous `timeout` field would be deprecated once this feature is beta." should read "stable" instead of "beta", since taskrun.timeout is now stable and we can't replace it with a beta feature.

Reply:
Sorry for squishing a lot of suggestions into one comment; it looks like the PipelineRun section still needs to be updated though.

IMO, it makes sense to have a "scheduling" option in `pipelinerun.spec.timeouts.taskruns` (since how long it takes to schedule a taskrun doesn't depend on the contents of the taskrun), but probably doesn't make sense to add `pipelinerun.spec.timeouts.execution`, because that will differ between taskruns. Either way, it's worth explaining the reasoning in the PipelineRuns section.